src/components/animated/_spotlight.less

```diff
@@ -0,0 +1,201 @@
+.sbtn.spotlight-btn{
+    /* @mrsidsat */
+    position: relative;
+    padding:0px;
+    color:black;
+    border:3px solid #0cbfcf ;
+    height:50px;
+    width:200px;
+    background-color: transparent;
+    overflow:visible !important;
+    cursor:pointer;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    box-shadow: inset 0px 0px 10px  #0fe0f5;  
+	.blue-btn{
+		border:3px solid blue;
+		box-shadow: inset 0px 0px 10px blue; 
+    }
+	.pink-btn{
+		border:3px solid pink;
+    	box-shadow: inset 0px 0px 10px pink;
+	}
+	.red-btn{
+		border:3px solid red;
+    	box-shadow: inset 0px 0px 10px red;  
+	}
+	.green-btn{
+		border:3px solid green;
+    	box-shadow: inset 0px 0px 10px green;
+	}
+	.yellow-btn{
+		border:3px solid yellow;
+    	box-shadow: inset 0px 0px 10px yellow;
+	}
+	.orange-btn{
+		border:3px solid orange;
+    	box-shadow: inset 0px 0px 10px orange;
+	}
+	.purple-btn{
+		border:3px solid purple;
+		box-shadow: inset 0px 0px 10px purple;
+	}
+	.black-btn{
+		border:3px solid black;
+		box-shadow: inset 0px 0px 10px black;
+	}
+	.white-btn{
+		border:3px solid  #0cbfcf;
+		box-shadow: inset 0px 0px 10px  #0cbfcf;  
+	  } 
+&:after{             
+    content:"";
+    border:3px solid #07dae9;
+    box-shadow: inset 0px 0px 10px #0fe0f5;
+    height:50px;
+    width:200px;
+    top:3px;
+    left:3px;
+    opacity: 1;
+    position: absolute;
+    z-index: -10;
+    background-color: transparent;
+	overflow:visible;
+	.blue-btn{
+		border:3px solid blue;
```

Comment:
these button colors will not work inside `:after` so please remove them. They are also unnecessary since they use the same code above. If you meant the selector `.spotlight-btn.blue-btn:after` for example, then put inside `&.blue-btn` above `&:after` and add whatever rules you want inside. Make sure to not repeat any code, so if it still uses the same code as above you should use mixins for that

Reply:
Okay I would make the necessary changes soon...